### DISCUSSION TITLE
Benefits for contributing to mdn docs??

### DISCUSSION THREAD

**Discussion by AbubakrChan** [mdn/mdn-community#578] [Content] [answered]
I know this is going to be helpful for other fellow developers, and I am really happy about this.
But what are additional benefits for contribution??
Thanks

**Comment by teoli2003** [comment, marked as answer]
Benefits for contribution to MDN:

- You improve MDN overall, even for you. The resource is better.
- Writing documentation is a skill that is not that easy. A lot of good developers I know are unable to write useful docs. Contributing lets you exercise this skill.
- Exercising your skills with git and GitHub (working with PRs, …). These tools are part of many modern toolchains and are used by many companies. Technical writers should be fluent with such tools.
- Writing about a subject means going to the bottom of a subject. Like when teaching, you learn a lot while doing this. There is no true expertise if you can't explain something with simple words.
- Build relationships with other participants in a project: people will review your PRs and interact with you, and you'll learn new ways of doing things or new points of view.
- Helping the community. It is a good thing.
- Fun. Because documenting the web is fun.
